ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -89,19 +86,19 @@ def _fetch_available_backends(logger):
                     for osds in stack.values():
```

Comment:
Actually it is required because the osd objects contain way less information than the local_stack.